translations/translate.bash

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# This script updates all translation .po files for every extension in the ADR project
+# The result translations are copied into this folder so that they can be shared with UNAIDS
+# Before running ensure that you have every repo checked out with latest development branch
+# To use this script: `adx bash ckan` followed by `bash /usr/lib/adx/adx_develop/translations/translate.bash`
```

Comment:
Can you rename the script to `translate.sh` so it's consistent with other scripts in the repo?